app/views/layouts/application.html.erb

```diff
@@ -16,11 +16,11 @@
 
   <body>
     
-    <%= render "shared/header" %>
+    <%= render "partials/header" %>
```

Comment:
I prefer shared, it's a little more descriptive as to what these partials are and do